### PR TITLE
Gs comment setup, create

### PR DESCRIPTION
**Pull request by gavinschriver**
# Description
added logic in request_handler.py for a POST to /comments with matching function in comment/request.py, also set up comment model 
Fixes # (issue) 
## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# How Has This Been Tested?
Please describe step-by-step the process that a reviewer can follow to adequately test this PR. 
- [x] `git fetch --all`
- [x] `git checkout gs-comment-setup-create`
- [ ] open up postman and fire off this sucker as a POST in the body (RAWWWW JSON format) to
`http://localhost:8088/comments` 
`{"content":"this is a greattt idea","timestamp": 3948699542 ,"post_id": 2, "user_id": 5}`
- [ ] make sure you get that new resource back with an ID and check your comment table for the hell of it to make sure its saving 

PART 2 ELECTRIC BOOGALOO:

- [ ] after making at least one comment thru whatsoever means you choose, run a GET request from postman to 
`http://localhost:8088/comments?post_id=<post_IdValueForACommentThatExists>` and make sure you get back an array of json comment objects w embedded user with first and last name and username

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by mgjeffries** [comment]
I think that we should go ahead and update the rare.db file to include the table for comments.

File: comments/request.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import sqlite3
+import json
+from models import Comment, User, Post
+
+# get_all_comments, #get_comments_by_post_id, #get_single_comment
```
Comment (mgjeffries) [inline comments/request.py:5]: This comment is not clear. Are these functions to add later?

File: rare_server.session.sql [hunk 2]
```diff
@@ -99,4 +99,20 @@ UPDATE post SET publication_time = 1318781876406
...
+
+CREATE TABLE `comment` (
+  `id` INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+  `content` INTEGER NOT NULL,
+  `timestamp` INTEGER NOT NULL,
+  `post_id` INTEGER NOT NULL,
+  `user_id` INTEGER NOT NULL,
+  FOREIGN KEY (`post_id`) REFERENCES `post` (`id`),
+  FOREIGN KEY (`user_id`) REFERENCES `user` (`id`)
+);
```
Comment (mgjeffries) [inline rare_server.session.sql:112]: This format matches the ERD. Great!

File: comments/request.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        WHERE c.post_id = ?
+        """, (post_id,))
+        comments = []
+        dataset = db_cursor.fetchall()
+        for row in dataset:
+            comment = Comment(row['id'], row['content'], row['timestamp'], row['post_id'], row['user_id'])
+            comments.append(comment.__dict__)
+            user = User(row['user_id'], row['first_name'], row['last_name'], row['username'], "", "")
+            comment.user = user.__dict__
+        return json.dumps(comments)
```
Comment (mgjeffries) [inline comments/request.py:44]: I like that you are expanding comments by user!

**Review by mgjeffries — APPROVED** [review]

**Review by FORDBA — APPROVED** [review]

**Pull request merged**